src/components/sbb-selection-panel/sbb-selection-panel.tsx

```diff
@@ -1,133 +1,101 @@
-import {
-  Component,
-  ComponentInterface,
-  h,
-  Element,
-  Event,
-  Host,
-  JSX,
-  Listen,
-  Prop,
-  State,
-  EventEmitter,
-} from '@stencil/core';
 import { CheckboxStateChange } from '../sbb-checkbox/sbb-checkbox.custom';
 import { RadioButtonStateChange } from '../sbb-radio-button/sbb-radio-button.custom';
 import {
   createNamedSlotState,
   HandlerRepository,
   namedSlotChangeHandlerAspect,
+  EventEmitter,
+  ConnectedAbortController,
 } from '../../global/eventing';
 import { InterfaceSbbSelectionPanelAttributes } from './sbb-selection-panel.custom';
```

Comment:
Since the "global inlining" of types has already started (not 100% sure it is), inline them in this PR pls

Reply:
Types removed and events moved into the component class.